### PR TITLE
docs(accessibility): init accessibility docs

### PR DESCRIPTION
**Pull request by nikkimk**
## What I did

1. Accessibility documentation

## Testing Instructions

1. Review [Accessibility](https://deploy-preview-1340--red-hat-design-system.netlify.app/accessibility/) docs

### CODE REVIEW

**Comment by brianferry** [comment]
I wonder if in our accessibility docs we should include links to other accessibility tools that we think help or not.  Or maybe instead of individual endorsements something like https://github.com/brunopulis/awesome-a11y ? vs linking directly to something like https://github.com/michelf/sim-daltonism

File: docs/accessibility/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Accessibility
+---
+
+Making content accessible is an opportunity to be creative, challenge convention, and be ahead of the curve when it comes to making the web a less hostile place.
+
+“The power of the Web is in its universality. Access by everyone regardless of disability is an essential aspect.” — Tim Berners-Lee, W3C Director and inventor of the World Wide Web
```
Comment (zeroedin) [inline docs/accessibility/index.md:7]: dog food `<rh-blockquote>` here maybe?

File: docs/accessibility/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+There are plenty more examples we could include. Essentially, accessibility takes _all users_ into account.
+
+Accessibility spectrum example
+
+| Type      | Situational | Temporary | Permanent |
+| --------- | ----------- | --------- | --------- |
+| Touch     | ...         | ...       | ...       |
+| Sight     | ...         | ...       | ...       |
+| Auditory  | ...         | ...       | ...       |
+| Sppech    | ...         | ...       | ...       |
```
Comment (zeroedin) [inline docs/accessibility/index.md:65]: typo

```Suggestion
| Speech    | ...         | ...       | ...       |
```

File: docs/accessibility/index.md (hunk 1 above)
Comment (zeroedin) [reply docs/accessibility/index.md]: Not blocking but would be nice

**Review by zeroedin — APPROVED** [review]
This is good to go just the two nits, one spelling typo

**Pull request merged**